app/assets/javascripts/discourse/app/raw-views/topic-list-header-column.js

```diff
@@ -45,4 +45,24 @@ export default EmberObject.extend({
 
     return name.join(" ");
   },
+
+  @discourseComputed
+  ariaPressed() {
+    if (this.sortable) {
```

Comment:
I think you just use computed property and:

```
ariaPressed: and("sortable", "iSorting")
```

https://api.emberjs.com/ember/release/functions/@ember%2Fobject%2Fcomputed/and